examples/flash_example/server.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+from functools import partial
+from typing import Any, Dict, Optional
+
+import flwr as fl
+from flwr.common.typing import Config
+from flwr.server.client_manager import SimpleClientManager
+from flwr.server.strategy import FedAvg
+
+from examples.models.cnn_model import Net
+from examples.utils.functions import make_dict_with_epochs_or_steps
+from fl4health.server.base_server import FlServer
+from fl4health.utils.config import load_config
+from fl4health.utils.metric_aggregation import evaluate_metrics_aggregation_fn, fit_metrics_aggregation_fn
+from fl4health.utils.parameter_extraction import get_all_model_parameters
+
+
+def fit_config(
+    batch_size: int,
+    n_server_rounds: int,
+    downsampling_ratio: float,
+    current_round: int,
+    local_epochs: Optional[int] = None,
+    local_steps: Optional[int] = None,
+) -> Config:
+    return {
+        **make_dict_with_epochs_or_steps(local_epochs, local_steps),
+        "batch_size": batch_size,
+        "n_server_rounds": n_server_rounds,
+        "downsampling_ratio": downsampling_ratio,
+        "current_server_round": current_round,
+    }
+
+
+def main(config: Dict[str, Any]) -> None:
+    # This function will be used to produce a config that is sent to each client to initialize their own environment
+    fit_config_fn = partial(
+        fit_config,
+        config["batch_size"],
+        config["n_server_rounds"],
+        config["downsampling_ratio"],
+        local_epochs=config.get("local_epochs"),
+        local_steps=config.get("local_steps"),
+    )
+
+    initial_model = Net()
+
+    # Server performs simple FedAveraging as its server-side optimization strategy
+    strategy = FedAvg(
```

Comment:
Perhaps I'm missing something here, but this doesn't look like it's using the FLASH strategy that you created?

Reply:
good catch! corrected this.